recipe/meta.yaml

```
@@ -39,6 +37,7 @@ outputs:
         - python
         - click >=7.*
```

```suggestion
        - click >=7.0
```
https://github.com/encode/uvicorn/blob/7dc027d5fb980c7bd52ab4611f3109a796cec974/pyproject.toml#L33